roles/jd-server/src/lib/job_declarator/mod.rs

```diff
@@ -145,7 +145,7 @@ impl JobDeclaratorDownstream {
                     .ok_or(Box::new(JdsError::ImpossibleToReconstructBlock(
                         "Txid found in jds mempool but transactions not present".to_string(),
                     )))?;
-                transactions_list.push(tx);
+                transactions_list.push(tx.0);
```

Comment:
Why did you change this?

Reply:
Since the hashmap structure is now `(txid, (tx, count))` instead of `(txid, tx)`, we used the first element of the inner tuple to reference `tx`.